refile.gemspec

```diff
@@ -34,9 +34,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "aws-sdk"
   spec.add_development_dependency "rack-test", "~> 0.6.2"
   spec.add_development_dependency "rails", "~> 4.1.8"
-  spec.add_development_dependency "sqlite3"
+  spec.add_development_dependency RUBY_PLATFORM == "java" ? "activerecord-jdbcsqlite3-adapter" : "sqlite3"
```

Comment:
Good idea! That's typically how we handle dev dependencies. Bundler has more robust support for multiple platform dependencies.